.github/workflows/flatpak.yml

```diff
@@ -19,35 +24,91 @@ env:
   YOUTUBE_SECRET_HASH: ${{ secrets.YOUTUBE_SECRET_HASH }}
 
 jobs:
-  flatpak_builder:
-    name: Bundle
+  generate_bundle:
+    name: Generate Flatpak Bundle
+    runs-on: [ubuntu-latest]
+    if: github.event_name != 'release'
+    container:
+      image: bilelmoussaoui/flatpak-github-actions:kde-5.15-21.08
+      options: --privileged
+    steps:
+      - name: 'Check for Github Labels'
+        if: github.event_name == 'pull_request'
+        shell: bash
+        run: |
+          LABELS_URL="$(echo ${{ github.event.pull_request.url }} | sed s'/pulls/issues/')"
+          LABEL_FOUND="$(curl -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" "${LABELS_URL}/labels" | sed -n 's/.*"name": "\(.*\)",/\1/p' | grep 'Seeking Testers' || true)"
+          if [ "${LABEL_FOUND}" = "Seeking Testers" ]; then
+            echo "SEEKING_TESTERS=1" >> $GITHUB_ENV
+          else
+            echo "SEEKING_TESTERS=0" >> $GITHUB_ENV
+          fi
+
+      - name: Checkout
+        uses: actions/checkout@v2.3.3
+        if: success() && (github.event_name != 'pull_request' || env.SEEKING_TESTERS == '1')
+        with:
+          submodules: 'recursive'
+
+      - name: Build Flatpak Manifest
+        uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@v3
```

Comment:
any reason for using v3 and not v4 or master like the flat-manager action?

Reply:
Because this is a cosmetics-only comit

Reply:
Added an extra commit changing that.